libs/template/presets/storefront/experience/pages/registration-page.ts

```diff
@@ -58,69 +49,94 @@ export const registrationPage: ExperienceComponent = {
           type: 'oryx-composition',
           options: {
             rules: [
+              { layout: CompositionLayout.List, colSpan: 2 },
+              {
+                query: { breakpoint: Size.Md },
+                width: '80%',
+                colSpan: 3,
+              },
               {
-                layout: CompositionLayout.Grid,
-                gap: '20px',
-                style: `grid-template-columns: 1fr 1fr`,
+                query: { breakpoint: Size.Lg },
+                gridColumn: 2,
+                colSpan: 2,
               },
             ],
           },
           components: [
             {
-              type: 'oryx-content-text',
-              options: {
-                rules: [{ colSpan: 2 }],
-              },
-              content: { data: { text: `<h5>Have an account?</h5>` } },
-            },
-            {
-              type: 'oryx-content-text',
-              options: {
-                rules: [{ width: '100%' }],
-              },
-              content: {
-                data: {
-                  text: `
-                      <oryx-button
-                        href="/login"
-                        type=${ButtonType.Outline}
-                        size=${ButtonSize.Md}
-                        style="width: 100%"
-                      >
-                        Login
-                      </oryx-button>
-                    `,
-                },
-              },
-            },
-            {
-              type: 'oryx-content-text',
+              type: 'oryx-composition',
               options: {
                 rules: [
                   {
-                    colSpan: 2,
-                    margin: '10px 0 0',
+                    layout: CompositionLayout.Grid,
```

Comment:
Please, don't use `CompositionLayout` we are going to deprecate it

Reply:
Thanks, fixed.